hail_scripts/v02/utils/elasticsearch_client.py

```diff
@@ -79,8 +80,10 @@ def export_table_to_elasticsearch(
                 this string in all field names. This replacement is not reversible (or atleast not
                 unambiguously in the general case) Set this to None to disable replacement, and fall back
                 on an encoding that's uglier, but reversible (eg. "." will be converted to "_$dot$_")
+            func_to_run_after_index_exists (function): optional function to run after creating the index, but before exporting any data.
```

Comment:
Would it be better to split out the index creation into another function? `export_table_to_elasticsearch` could by default call it before exporting any data, but in the case where you need to do something in between, you could call them separately.

Reply:
I don't have a strong opinion on this. These methods did get too big in v01. 
I need to make one more change to the v01 method now (to allow parent/child relations to have custom names). Beyond that though, if there's a particular way you want these methods (as well as the computed_fields ones ) to be structured, do you think you could port them (while I finish up the reference data scripts)?